dev/META6.json

```diff
@@ -11,12 +11,15 @@
   "depends": [
     "Cro::HTTP",
     "Cro::HTTP::Session::Pg",
+    "Cro::HTTP::Test",
```

Comment:
Not a big issue, but strictly belongs in `test-depends`.

Reply:
Moved together with the other test related modules.